Resources/Prototypes/Catalog/uplink_catalog.yml

```
@@ -208,6 +210,8 @@
     Telecrystal: 150 # goob
   categories:
   - UplinkWeaponry
+  conditions:
```

Indicate funky change.
```suggestion
  conditions: # Funky war ops whitelisting
```